react/components/Feed/index.js

```diff
@@ -33,6 +35,14 @@ export default class Feed extends PureComponent {
     hasMore: true,
   }
 
+  getContext = (data) => {
+    if (!data) return [];
+
+    const { me: { feed: { groups } } } = data;
+    const objects = flatten(map(groups, group => group.objects));
+    return filter(objects, object => (object.__typename !== 'Channel' && object.__typename !== 'Group' && object.__typename !== 'User'));
```

Comment:
https://github.com/aredotna/ervell/blob/master/react/components/Cell/components/Konnectable/index.js#L99 also?

Reply:
Groups and users are on the feed, didn't seem like something that belonged in Konnectable?

Reply:
I don't know I don't remember any of how I got it working. Realistically it doesn't belong in either place